### PR TITLE
Bluegiga support

### PR DESCRIPTION
**Pull request by paulianttila**
Add support for Bluegiga adapters via pygatt

### CODE REVIEW

**Comment by ttu** [comment]
Thanks! Very nice addition. Can't test this myself, but will check the code 👍 

No worries if you haven't tested all corner cases, you can always make a fix later.

File: ruuvitag_sensor/adapters/bluegiga.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        Attributes:
+           device (string): BLE device (default auto)
+        """
+
+        if bt_device:
+            adapter = pygatt.BGAPIBackend(bt_device)
+        else:
+            adapter = pygatt.BGAPIBackend()
+
+        reset = False if os.environ.get('BLUEGIGA_RESET', '').upper() == 'FALSE' else True
```
Comment (ttu) [inline ruuvitag_sensor/adapters/bluegiga.py:95]: Add `BLUEGIGA_RESET` to README

Comment (paulianttila) [reply ruuvitag_sensor/adapters/bluegiga.py:95]: Done

File: ruuvitag_sensor/ruuvi.py [hunk 2]
```diff
@@ -13,6 +13,17 @@
 if os.environ.get('RUUVI_BLE_ADAPTER') == 'Bleson':
     from ruuvitag_sensor.adapters.bleson import BleCommunicationBleson
     ble = BleCommunicationBleson()
+elif os.environ.get('RUUVI_BLE_ADAPTER') == 'Bluegiga':
+    def get_raw_bluegiga(raw, data_format):
+        # Bluegiga drops FF from the raw data and it is required for DF 3 and 5
+        # TODO: Move convert_data to adaptor specific code
+        if data_format in (2, 4):
```
Comment (ttu) [inline ruuvitag_sensor/ruuvi.py:20]: Could you try to add something similar to Bluegiga adapter?
https://github.com/ttu/ruuvitag-sensor/blob/407867b65a75d84f66f2035e1fdc5e57fcc518b2/ruuvitag_sensor/adapters/bleson.py#L51

Unfortunately can't try to do that myself. Will merge this PR soon and doesn't matter if you have time to add this or not. Would just be nice to get this parsing hack completely away from adapter decision code as in Bleson's case this is now done in the adapter code.
